coverage-tests.js

```diff
@@ -2288,4 +2288,25 @@ page: 'HelloWorld',
   }
 })
 
+test('Should return exception in TestResultsSummary', {
+  page: 'AdoptedStyleSheets',
+  vg: true,
+  config: {
+    browsersInfo: [
+      {name: 'firefox', width: 640, height: 480},
+    ],
+  },
+  test({eyes, assert}) {
+    eyes.open({appName: 'Applitools Eyes SDK', viewportSize})
+    eyes.check({isFully: false})
+    assert.throws(() => void eyes.close())
+    // TODO assert test is aborted
+    eyes.open({appName: 'Applitools Eyes SDK', viewportSize})
+    eyes.check({isFully: false, visualGridOptions: {polyfillAdoptedStyleSheets: true}})
+    eyes.close(false)
+    const summary = eyes.runner.getAllTestResults(false)
+    assert.equal( summary.getAllResults()[0]._container.exception.message.substring(0,27), `failed to render screenshot`)
```

Comment:
What is the full message?

Reply:
@amitzur 
`failed to render screenshot 39ffba28-77ec-4f56-aee0-0bb5acd5abd6 got error: This page's DOM has a feature, Adopted Stylesheets (also known as Constructible Stylesheets),
that is currently supported only in Chrome (you can find information here: https://wicg.github.io/construct-stylesheets/).
This means that this page will look different if used in browsers other than Chrome (e.g. Firefox).
It may be that your web code polyfills this feature in other browsers. In that case, please
set "visualGridOptions.polyfillAdoptedStyleSheets" to true when checking the page,
and the Visual Grid will polyfill this feature for you too. If you would like to see how the page
looks in other browsers without the polyfill, you can set the "visualGridOptions.polyfillAdoptedStyleSheets" to false.
But until you set this flag "visualGridOptions.polyfillAdoptedStyleSheets" to true or false, this check will fail.`

Reply:
oh wow :)